comp/forwarder/defaultforwarder/blocked_endpoints.go

```diff
@@ -15,9 +15,50 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/backoff"
 )
 
+// TimeNow useful for mocking
+var TimeNow = time.Now
```

Comment:
Rather than mocking time, consider accepting time as an argument from the caller.